homeassistant/components/fan/dyson.py

```diff
@@ -233,3 +236,11 @@ def speed_list(self: ToggleEntity) -> list:
     def supported_features(self: ToggleEntity) -> int:
         """Flag supported features."""
         return SUPPORT_OSCILLATE | SUPPORT_SET_SPEED
+
+    @property
+    def state_attributes(self) -> dict:
+        """Return optional state attributes."""
+        attributes = super().state_attributes
```

Comment:
This isn't needed. `device_state_attributes` will be merged with `state_attributes` in the entity base class.

Reply:
Oh okay I see!